pkg/util/mount/mount_windows.go

```diff
@@ -358,10 +358,23 @@ func (mounter *SafeFormatAndMount) formatAndMount(source string, target string,
 	glog.V(4).Infof("Attempting to formatAndMount disk: %s %s %s", fstype, source, target)
 
 	if err := ValidateDiskNumber(source); err != nil {
-		glog.Errorf("azureMount: formatAndMount failed, err: %v\n", err)
+		glog.Errorf("diskMount: formatAndMount failed, err: %v\n", err)
 		return err
 	}
 
+	if len(fstype) == 0 {
+		// Use 'NTFS' as the default
+		fstype = "NTFS"
+	}
+
+	// format disk if it is unformatted(raw)
+	cmd := fmt.Sprintf("Get-Disk -Number %s | Where partitionstyle -eq 'raw' | Initialize-Disk -PartitionStyle MBR -PassThru", source)
+	cmd += fmt.Sprintf(" | New-Partition -AssignDriveLetter -UseMaximumSize | Format-Volume -FileSystem %s -Confirm:$false", fstype)
```

Comment:
I'm not familiar with windows commands, but does this skip formatting if the device is already formatted?

Reply:
yes, since it's using `Where partitionstyle -eq 'raw'` to filter out the formatted disk, and if there is no raw disk, it won't return error which is expected in the code logic

Reply:
nit: can these two lines be consolidated?

Reply:
fixed